CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(APPLE)
 	set_property(DIRECTORY
 		APPEND
 		PROPERTY COMPILE_DEFINITIONS SC_DARWIN)
+	set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install")
```

Comment:
you are overriding both the cmake defaults and the command line arguments passed to cmake: if a user sets the install prefix, you simply overwrite what the user requests.

fwiw, on osx there is no reason to use "install" anyway, as app bundles are self contained and osx isn't really fhs-compliant, anyway